docker-compose.yml

```diff
@@ -35,14 +41,23 @@ services:
     links:
       - mongo
     volumes:
-      - ./bin:/app/bin
-      - ./config:/app/config
-      - ./endpoints:/app/endpoints
-      - ./lib:/app/lib
-      - ./models:/app/models
-      - ./parsers:/app/parsers
-      - ./routes:/app/routes
-      - ./package.json:/app/package.json
+      - ./shared:/app/shared
+      - ./backend:/app/backend/
+    depends_on:
+      - mongo
+    working_dir: /app/backend
+    command: npm run watch
+
+  ycai:
+    image: yttrex_base
+    environment:
+      - NODE_ENV=development
+    volumes:
       - ./tsconfig.json:/app/tsconfig.json
-    working_dir: /app
+      - ./shared:/app/shared
+      - ./YCAI:/app/YCAI
+      - ./YCAI/build:/app/YCAI/build
```

Comment:
@djfm here I'm binding the `build` output to docker volume, so that dockerized process outputs in your local `build` dir

Reply:
thanks, I had missed that